src/modules/search/lookup-service.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/**
+ * lookup utility used by searchIntent
+ */
+module.exports = {
+  /**
+   * Retrieves explanation for one key word using localized search function
+   * @param {string} key - word to search
+   * @param {Function} localizedSearchFcn - translation function, takes options object specifying falsy return value when translation was not found
+   */
+  search: (key, localizedSearchFcn) => new Promise(resolve => {
+    resolve(localizedSearchFcn(key, { defaultValue: null })); // cannot use default value undefined
```

Comment:
Took me while to understand the purpose of the default value so I would change the comment to something like: `// Resolving as default value: `null` instead of 'key' if it doesn't exist in dictionary`